thentos-adhocracy/src/Thentos/Adhocracy3/Types.hs

```diff
@@ -72,7 +72,7 @@ instance EmptyDB DB where
 
 instance DB `Core.Extends` Core.DB where
     focus f (DB db) = DB <$> f db
-    asDBThentosError = ThentosA3ErrorCore
+    thentosErrorFromParent = ThentosA3ErrorCore
```

Comment:
I agree.  Which error or FIXME did trigger this comment?  (Can't find anything in the code around it.)